app/vlinsert/jsonline/jsonline.go

```diff
@@ -89,11 +104,11 @@ func readLine(lr *insertutil.LineReader, timeField string, msgFields []string, l
 	defer logstorage.PutJSONParser(p)
 
 	if err := p.ParseLogMessage(line); err != nil {
-		return true, fmt.Errorf("cannot parse json-encoded line: %w; line contents: %q", err, line)
+		return true, fmt.Errorf("%s; line contents: %q", err, line)
 	}
 	ts, err := insertutil.ExtractTimestampFromFields(timeField, p.Fields)
 	if err != nil {
-		return true, fmt.Errorf("cannot get timestamp from json-encoded line: %w; line contents: %q", err, line)
```

Comment:
This change is needed to avoid duplicating error messages like:
`cannot get timestamp from json-encoded line: cannot parse timestamp from field "_time": cannot parse unix timestamp from "foobar"`